service/query/addresses.js

```diff
@@ -19,30 +19,30 @@ module.exports = function(app) {
 		// Request basic account information
 		axios({
 			method: 'get',
-			url: `${constants.algodurl}/account/${address}`, // Request transaction details endpoint
+			url: `${constants.algodurl}/v2/accounts/${address}`, // Request transaction details endpoint
 			headers: {'X-Algo-API-Token': constants.algodapi}
 		}).then(response => {
 			let result = response.data; // Set data to result
 
 			axios({
 				method: 'get',
-				url: `${constants.algodurl}/account/${address}/transactions?max=25`,
-				headers: {'X-Algo-API-Token': constants.algodapi}
+				url: `${constants.algoIndexerUrl}/v2/accounts/${address}/transactions?limit=25`,
+				headers: {'X-Indexer-API-Token': constants.algoIndexerToken}
 			}).then(async resp => {
 				let rounds = [];
 
 				for (let i = 0; i < resp.data.transactions.length; i++) {
-					rounds.push(resp.data.transactions[i].round);
+					rounds.push(resp.data.transactions[i]['confirmed-round']);
```

Comment:
Also need some pointers on how to replicate the issues. And full disclosure I'm using Sandbox to create transactions, because I keep encountering problems with the indexer I set up locally using the snapshot approach.

Reply:
using the sandbox is perfectly fine. 
in order to replicate this one you can search for any address (e.g. EVBBO2GHECBQQFM2CGCZ2LZPUUUBWZYSXYSTXBXPHDITC6QSCOEC7DJ4JU) right after you start the sandbox.